README.md

```diff
@@ -392,22 +392,46 @@ your-secretRootKey-value:
 
 [Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
 
-## Automatic Credentials Rotation
+## Automatic Service Binding Rotation
+
+You can rotate your service binding to ensure credentials are reset from time to time. Rotating means regularly generating new credentials while the old ones are discarded. 
+
 To enable automatic credentials rotation, you need to set the following parameters of the `credentialsRotationPolicy` field in the `spec` field of the `ServiceBinding` resource:
 
-- `enabled` - Whether the credentials rotation option is enabled. Default value is false.
-- `rotationFrequency` - Indicates the frequency at which the credentials rotation is performed.
-- `rotatedBindingTTL` - Indicates for how long to keep the rotated `ServiceBinding`.
 
-Valid time units for `rotationFrequency` and `rotatedBindingTTL` are: "ns", "us" or ("µs"), "ms", "s", "m", "h".</br></br>
-`status.lastCredentialsRotationTime` indicates the last time the `ServiceBinding` secret was rotated.</br>
-Please note that `credentialsRotationPolicy` evaluated and executed during [control loop](https://kubernetes.io/docs/concepts/architecture/controller/) which runs on every update or during
+| Parameter         | Type     | Description                                      |   Valid Values  |
+|:-----------------|:---------|:----------------------------------------------------|:---------------------|
+| `enabled` | bool | Whether the credentials rotation option is enabled.    |                                                                                                                                                          |
+| `rotationFrequency` | number | Indicates the interval between credential rotations. |      "ns", "us" or ("µs"), "ms", "s", "m", "h" |                                                                                                                                                                |
+| `rotatedBindingTTL`   |  number | Specifies for how long to keep the rotated `ServiceBinding`.   |      "ns", "us" or ("µs"), "ms", "s", "m", "h" |   
+
+
+**Recommendation** 
+
+Set the TTL shorter than the rotation frequency. This gives you the opportunity to avoid the transition period timely inform your users of new credentials before the binding's validity expires. 
+
+
+
+If your TTL is longer than the rotation frequency,  the following occurs:
+
+There is a transition period during which the rotated binding is still valid. Namely, there are two (or more) `ServiceBinding` objects: the original and the rotated one (holds the `services.cloud.sap.com/stale` label). 
+The rotated one is deleted once the `rotatedBindingTTL` duration elapses.
+
```

Comment:
The TTL provides an opportunity to inform your users of new credentials before the binding's validity expires. To optimize this process, set the TTL to be shorter than the rotation frequency. This eliminates the need to back up two sets of credentials simultaneously.